### PR TITLE
Stats affecting NSA

### PR DESCRIPTION
**Pull request by SirRichardFrancis**
## About The Pull Request

Neural system accumulation threshold would scale with COG stat, adding 1 point to NSA limit for each 3 points of COG.

image

## Why It's Good For The Game

Most useless stat finally good for something.

## Changelog
:cl:
add: COG affecting nsa_threshold
/:cl:

### CODE REVIEW

**Comment by MadmanMartian** [comment]
Siding any NSA-heavy chem with the COG boosting chems is gonna be a new strat

**Comment by SirRichardFrancis** [comment]
> Siding any NSA-heavy chem with the COG boosting chems is gonna be a new strat

Finally someone gonna make those COG boosting chems.

File: code/modules/reagents/reagents/medicine.dm [hunk 1]
```diff
@@ -682,19 +682,19 @@
 	metabolism = REM/2
 
 /datum/reagent/medicine/detox/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
-	if(M.metabolism_effects.nsa_threshold == initial(M.metabolism_effects.nsa_threshold))
-		M.metabolism_effects.nsa_threshold += rand(20, 60)
+	if(M.metabolism_effects.nsa_threshold_base == initial(M.metabolism_effects.nsa_threshold_base))
```
Comment (Evankhell561) [inline code/modules/reagents/reagents/medicine.dm:685]: this is problematic and gives problems with respect to perks.

Comment (SirRichardFrancis) [reply code/modules/reagents/reagents/medicine.dm:685]: It is, but this chem is not my invention, it's been there before, and I just made sure that it works with new vars.

Comment (Evankhell561) [reply code/modules/reagents/reagents/medicine.dm:685]: I understand, this is just a comment and not a request for changes anyway.

**Review by TheShown911 — APPROVED** [review]

**Pull request merged**